Mathlib/Algebra/Order/GroupWithZero/WithZero.lean

```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kevin Buzzard
 -/
 import Mathlib.Algebra.Order.GroupWithZero.Canonical
-import Mathlib.Algebra.Order.GroupWithZero.Unbundled.Basic
```

Comment:
Probably this should be reverted too, we don't tend to remove imports besides the ones that `shake` tells us to.